modules/javafx.graphics/src/test/java/test/javafx/scene/layout/RegionTest.java

```diff
@@ -1248,4 +1251,33 @@ public void clearDirty() {
         NodeHelper.syncPeer(r);
         assertFalse(peer.isClean());
     }
+
+    // Test for JDK-8199592
+    @Test public void snappingASnappedValueGivesTheSameValueTest() {
```

Comment:
code comment still pointing to old issue :)

Reply:
Fixed